index.html

```diff
@@ -13,14 +13,23 @@
                     <h3>Welcome</h3>
                 </a>
                 <nav class="menu">
-                    <img src="assets/img/menu.svg" alt="menu link">
+                    <img src="assets/img/menu.svg" alt="menu link" id="hamburger">
                 </nav>
                 <nav class="desktop-toolbar">
                     <li class="desktop-toolbar__li"><a href="#work-title">Portfolio</a></li>
                     <li class="desktop-toolbar__li"><a href="#about-myself">About</a></li>
                     <li class="desktop-toolbar__li"><a href="#form">Contact</a></li>
                 </nav>
             </div>
+            <nav class="header-nav-mobile hidden">                
```

Comment:
- [ ] [OPTIONAL] Kindly prevent the user from scrolling to the page when the menu is opened(make the menu fixed) for a better user experience.